backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/GwtClipboard.java

```diff
@@ -36,7 +36,7 @@ public String getContents () {
 	@Override
 	public void setContents (String content) {
 		this.content = content;
-		if (requestedWritePermissions) {
+		if (requestedWritePermissions || GwtApplication.agentInfo().isFirefox()) {
```

Comment:
That's not related to the accelerometer. But because I read [this](https://stackoverflow.com/questions/53147944/firefox-permission-name-member-of-permissiondescriptor-camera-is-not-a-vali) stackoverflow question, I recognized that querying the permission for the clipboard raises also an error in Firefox. I did not recognize the error as I made the PR because the implementation works fine, even with the error. I think it's not beautiful to get the error so I added the fix